src/DynamoCoreWpf/UI/InOutPortPanel.cs

```diff
@@ -48,8 +54,13 @@ protected override Size MeasureOverride(Size constraint)
                     // Default behavior of getting each child to measure.
                     child.Measure(constraint);
                 }
-                catch(System.Exception)
+                catch(XamlParseException e)
                 {
+                    if(e != null)
```

Comment:
Is this null check really required, won't `e` always be non-null?

Reply:
https://stackoverflow.com/questions/5634417/caught-exception-is-null-itself/27902976

Reply:
yeah, I thought I did this for a good reason, but it was a hold over from checking inner exceptions, I will remove unless theres something I'm missing.

Reply:
I do not think it is useful here

Reply:
It looks like it can be `null `if there are multiple `catch `blocks using the same variable name from that StackOverflow page.

Reply:
You can check for null in the Analytics.Core APIs